test/e2e/env/env.go

```diff
@@ -76,3 +83,20 @@ func (e TestEnv) GetLastSeenSlackMessage() string {
 	}
 	return ""
 }
+
+// SetupFakeWebhook create fake Slack server to mock Slack
+func (e *TestEnv) SetupFakeWebhook() {
+	s := webhooktest.NewTestServer()
+	go s.Start()
+
+	e.WebhookServer = s
+}
+
+// GetLastReceivedPayload return last message received by fake webhook server
+func (e TestEnv) GetLastReceivedPayload() utils.WebhookPayload {
+	allSeenMessages := e.WebhookServer.GetReceivedPayloads()
+	if len(allSeenMessages) != 0 {
+		return allSeenMessages[len(allSeenMessages)-1]
+	}
+	return utils.WebhookPayload{}
```

Comment:
If no message is there it should `nil` I guess instead of empty payload. Please let me know your opinion as well on this.

Reply:
> If no message is there it should nil

as it is returning the object struct itself, it is made to return the Zero Value of the struct(empty payload). for returning nil we might need to change it as a pointer.

Reply:
it is good to use pointers instead of passing the object directly.
shall we change it..? or lets leave this as is for now..? 
@PrasadG193 thoughts..?

Reply:
@aananthraj let's use pointers if that makes sense. We might need to modify return type of `GetLastSeenSlackMessage` also have consistency